server/middleware/populateCurrentUser.test.ts

```diff
@@ -0,0 +1,107 @@
+import { Request, Response } from 'express'
+import { createMockUserService } from '../services/testutils/mocks'
+import populateCurrentUser from './populateCurrentUser'
+
+const userService = createMockUserService()
+
+let req: Request
+let res: Response
+const next = jest.fn()
+
+describe('populateCurrentUser', () => {
```

Comment:
Adding test coverage for `populateCurrentUser` middleware because default file from the template changed.